src/System.Net.WebSockets.Client/src/System/Net/WebSockets/ManagedWebSocket.cs

```diff
@@ -463,15 +464,20 @@ private int WriteFrameToSendBuffer(MessageOpcode opcode, bool endOfMessage, Arra
             {
                 // We need to know where the mask starts so that we can use the mask to manipulate the payload data,
                 // and we need to know the total length for sending it on the wire.
-                int maskOffset = WriteHeader(opcode, _sendBuffer, payloadBuffer, endOfMessage, useMask: true);
-                headerLength = maskOffset + MaskLength;
+                maskOffset = WriteHeader(opcode, _sendBuffer, payloadBuffer, endOfMessage, useMask: true);
+                headerLength = maskOffset.Value + MaskLength;
```

Comment:
Slightly cheaper as GetValueOrDefault; we know it has a value s we just assigned one.